src/components/NavBar/NavItems.tsx

```diff
@@ -57,32 +69,35 @@ function NavItems({ classNameValue, isLoggedIn }: NavItemsProps) {
 							<>
 								<NavigationMenuTrigger>{item.name}</NavigationMenuTrigger>
 								<NavigationMenuContent>
-									<ul>
+									<ul className='grid gap-3 p-4 md:w-[400px] lg:w-[500px] lg:grid-cols-[.75fr_1fr]'>
```

Comment:
Please remove this styling - I've fixed the styling in my latest PR.

Reply:
Done